pygsp/graphs/fourier.py

```diff
@@ -88,26 +103,49 @@ def compute_fourier_basis(self, recompute=False):
         True
 
         """
+        if n_eigenvectors is None:
+            n_eigenvectors = self.N
 
-        if hasattr(self, '_e') and hasattr(self, '_U') and not recompute:
+        if hasattr(self, '_e') and hasattr(self, '_U') and not recompute \
+                and n_eigenvectors <= len(self.e):
```

Comment:
I'm not a fan of the `\` notation for line continuation. Can we use parentheses?